crates/virtio-queue-ser/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v0.4.1
+- Update the virtio-queue dependency to v0.7.1. This has not breaking changes.
```

Comment:
s/This has not breaking changes./This has no API breaking changes/?

Reply:
Done.